website/build.gradle

```diff
@@ -171,13 +172,17 @@ task publishWebsite << {
     // Cannot authenticate to the default github uri, so specify gitbox.
     git.remote.add(name: 'website-publish',
                    url: 'https://gitbox.apache.org/repos/asf/beam.git',
-                   pushRefSpecs: ['refs/heads/asf-site'])
+                   pushRefSpecs: ['asf-site'])
   }
   // Because git.push() fails to authenticate, run git push directly.
+  git.fetch()
```

Comment:
I believe this should have a `--all` parameter otherwise it only fetches the `origin` remote: https://git-scm.com/docs/git-fetch